docker/README.md

```diff
@@ -1,20 +1,35 @@
-The (Provided Dockerfile)[./Tangle.Dockerfile] is used as a base file to build different Docker images, the standalone node and the collator node.
+# Tangle Standalone Docker
```

Comment:
Could we remove standalone? I do not think we have other nodes, like a parachain one.